server/controllers/get_session_details.js

```diff
@@ -6,18 +6,15 @@ const getSessionResponsesNumberQuery = require("./../database/queries/get_sessio
 
 const GetSessionDetails = async (req, res) => {
   const { sessionId, sessionType } = req.params;
-  console.log(req.params,"param");
-  
+
   const promises = [
     getSessionQuestionsQuery(sessionType),
     getSessionAttendeesQuery(sessionId),
-    getSessionResponsesNumberQuery(sessionId),
+    getSessionResponsesNumberQuery(sessionId, sessionType),
   ];
   Promise.all(promises)
     .then(details => res.json(details))
-    .catch((err) => {
-      console.log(err);
```

Comment:
Why is the error logging being removed?

If this is because a request for a non-existent session ID is throwing an error, maybe it should not throw an error for this? 500 response should be for a server error caused by bad server logic, not a bad user request